ember-bootstrap/addon/helpers/bs-default.ts

```diff
@@ -0,0 +1,5 @@
+export default function bsDefaultHelper<PRIMARY, SECONDARY>(
```

Comment:
Please have a look at https://github.com/ember-bootstrap/ember-bootstrap/blob/2f0049c23d56a0b671f46981fb808ce7bcbe1797/ember-bootstrap/addon/helpers/bs-default.ts for a more advanced implementation, which does a null check as well.